aswf-vfx-base/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM centos/devtoolset-6-toolchain-centos7
+LABEL com.vfxplatform.version="CY2019"
+
+USER root
+
+RUN yum install -y \
+    bzip2-devel \
+    file \
+    make \
+    python-devel \
+    which \
+    zlib-devel \
+    git \
+    ca-certificates \
+    ssh \
+    sudo \
+    openssl-devel \
+    libxcb libxcb-devel \
+    libXrender libXrender-devel \
+    xcb-util-wm xcb-util-wm-devel \
+    xcb-util xcb-util-devel \
+    xcb-util-image xcb-util-image-devel \
+    xcb-util-keysyms xcb-util-keysyms-devel \
+    pulseaudio-libs-devel \
+    gperf \
+    ruby \
+    libcap-devel \
+    libXtst-devel \
+    pciutils-devel \
+    bison \
+    libv4l libv4l-devel \
+    libcap-devel \
+    mesa-libGL-devel \
+    mesa-libEGL-devel \
+    libxml2 libxml2-devel \
+    libxslt libxslt-devel \
+    nss nss-devel \
+    dbus dbus-devel \
+    fontconfig fontconfig-devel \
+    freetype freetype-devel \
+    ncurses ncurses-devel \
+    readline readline-devel \
```

Comment:
A few more yum packages missing from this list:

* doxygen
* pkgconfig
* cppunit-devel

Reply:
Oh and for Houdini:

* csh
* java-1.8.0-openjdk